its/ruling/src/test/resources/expected/python-S2737.json

```diff
@@ -0,0 +1,47 @@
+{
+'project:buildbot-0.8.6p1/buildbot/changes/bonsaipoller.py':[
+117,
+132,
+],
+'project:buildbot-0.8.6p1/contrib/bzr_buildbot.py':[
+264,
+],
+'project:buildbot-0.8.6p1/contrib/coverage2text.py':[
+84,
+],
+'project:django-2.2.3/django/contrib/admin/views/main.py':[
+422,
```

Comment:
To me, this looks like a FP which is rather well explained by the comments. I believe that we should exclude `except` clauses which are followed by other clauses which may catch the same exception.